src/containers/quiz/quiz-question/Question.tsx

```diff
@@ -47,110 +48,97 @@ const QuizQuestion: FC<QuizQuestionProps> = ({
   handleNonInputValueChange
 }) => {
   const { t } = useTranslation()
-
-  const { isMultipleChoice, isOpenAnswer } = determineQuestionType(
-    question.type
+  const { userRole } = useAppSelector((state) => state.appMain)
+  const { attemptId = '' } = useParams()
+  const initialIsCorrect = isCorrectAnswer(question, value)
```

Comment:
Please, rename a function since it's a **function**, not a **variable**
```suggestion
  const initialIsCorrect = checkAnswerCorrectness(question, value)
```